app/views/registration_wizard/choose_private_childcare_provider.html.erb

```diff
@@ -14,24 +18,49 @@
     <%= form_with model: @form, url: registration_wizard_form_url(@form), scope: 'registration_wizard', method: :patch do |f| %>
       <%= f.govuk_error_summary %>
 
-      <div class="npq-js-hidden">
-        <%= f.govuk_text_field :institution_name,
-                               width: "full",
-                               label: { text: "Enter your nursery name", class: "govuk-label govuk-label--s" } %>
-      </div>
+      <% if @form.display_no_javascript_fallback_form? %>
+        <% if @form.possible_institutions.blank? %>
+          <%= f.govuk_text_field :institution_name,
+                                 width: "full",
+                                 label: { text: question, size: "xl", tag: "h1" },
+                                 hint: { text: hint } %>
+        <% else %>
+          <%= f.govuk_radio_buttons_fieldset(:institution_identifier, legend: { size: 'xl', text: js_fallback_question, tag: "h1" }) do %>
+            <% @form.possible_institutions.each do |institution| %>
+              <%= f.govuk_radio_button :institution_identifier,
+                                       institution.identifier,
+                                       label: { text: [institution.urn, institution.name].compact.join(" - ") },
+                                       hint: { text: institution.address_string },
+                                       link_errors: true %>
+            <% end %>
```

Comment:
I think we just want `link_errors: true` on the first radio here:

```suggestion
            <% @form.possible_institutions.each_with_index do |institution, count| %>
              <%= f.govuk_radio_button :institution_identifier,
                                       institution.identifier,
                                       label: { text: [institution.urn, institution.name].compact.join(" - ") },
                                       hint: { text: institution.address_string },
                                       link_errors: count.zero? %>
            <% end %>
```

Reply:
I've made that change and also updated the other templates that used the same system. :)